### DISCUSSION TITLE
Model name details

### DISCUSSION THREAD

**Discussion by cachett-ML** [mlfoundations/open_clip#87] [Q&A]
Hi,

Where can we find the details behind your model naming?

Best,
Theo

**Comment by rwightman** [comment]
@cachett-ML moving to discussions...

All model details are in the configs https://github.com/mlfoundations/open_clip/tree/main/src/open_clip/model_configs

Pretrained model tags are based on the dataset they were trained on (names should be self evident), any `exx` suffix such as _32 refers to the epoch the checkpoint was from in training...
